labgrid/pytestplugin/fixtures.py

```diff
@@ -60,49 +60,47 @@ def env(request):
         pytest.skip("missing environment config (use --lg-env)")
 
     if request.config.pluginmanager.hasplugin('junitxml'):
```

Comment:
According to the source code at [pytest](https://docs.pytest.org/en/stable/_modules/_pytest/junitxml.html#record_testsuite_property), this should no longer be necessary, the fixture will return a no-op function in this case.